fpdcc/ccfp-asset-dashboard
# bring in ui polish with latest code + add interactions between map and table

## Overview
This PR has a ton of UI changes for issue #125. There are a lot of CSS changes and UI element tweaks. 

It also includes some React code changes for #120 and #96. The changes let a user select a row in the table which will open a popup for that row's geometry and allow the user to save that geometry. Similarly, a user can select a single geometry and save the geometry.

### Demo
![Screen Shot 2022-02-16 at 3 56 30 PM](https://user-images.githubusercontent.com/38969506/154364280-e2ac7c9b-5c6a-49c3-a688-6d9816f9c1e5.png)
![Screen Shot 2022-02-16 at 3 56 41 PM](https://user-images.githubusercontent.com/38969506/154364287-60a14bae-aff6-4b03-ba4d-296d126a9edb.png)
![Screen Shot 2022-02-16 at 3 57 06 PM](https://user-images.githubusercontent.com/38969506/154364291-352aec80-40ba-44f5-8f11-7431b3b72c71.png)
![Screen Shot 2022-02-16 at 3 57 36 PM](https://user-images.githubusercontent.com/38969506/154364294-aa282bb4-f70a-4f0d-83b5-962f96a4cd6e.png)
![Screen Shot 2022-02-16 at 3 57 53 PM](https://user-images.githubusercontent.com/38969506/154364301-4b77ed6f-8f3e-4e53-bbe2-4d53d3b1e742.png)

## Notes
At a high level, made these UI changes:
- widened the container that holds all the form elements
- branding: changed green for the header, added their logo, and used their icons
- changed all the links and buttons from dark green to light green
- most significant styling changes:
  - phase add / edit form
  - GIS tool

## Testing Instructions
- Review app: https://ccfp-asset-d-ui-polish-5fk6dqu.herokuapp.com
- Login with `ccfp-asset-dashboard test user` credentials in lastpass
- See the changes listed in the notes and make sure they look ok.

to test the map changes:
- click on a row in the table and confirm that the popup opens and you can save the selected asset
- click on an asset in the map and confirm that the popup opens and you can save the selected asset
